health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/PopInboxTable.js

```
@@ -95,6 +95,20 @@ const PopInboxTable = ({ ...props }) => {
               fieldValue
             );
           },
+          sortFunction: (rowA, rowB) => {
+            const fieldA = rowA.additionalFields.find((f) => f.key === field.key);
+            const fieldB = rowB.additionalFields.find((f) => f.key === field.key);
+          
+            const valueA = parseFloat(fieldA?.value || 0); // Converting to number, default to 0 if undefined
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)